include/jsoncons/json_options.hpp

```diff
@@ -365,7 +365,7 @@ class basic_json_encode_options : public virtual basic_json_options_common<CharT
           line_length_limit_(other.line_length_limit_),
           new_line_chars_(std::move(other.new_line_chars_)),
           after_key_chars_(std::move(other.after_key_chars_)),
-          indent_chars_(std::move(other.indent_chars))
+          indent_chars_(std::move(other.indent_chars_))
```

Comment:
How did this compile ? https://github.com/dragonflydb/dragonfly/actions/runs/12808735188/job/35712101156